.github/workflows/release.yml

```diff
@@ -146,21 +131,12 @@ jobs:
       contents: write
 
     steps:
-      - name: "Create temporary app token"
-        uses: actions/create-github-app-token@v1
-        id: app-token
-        with:
-          app-id: ${{ vars.APP_ID }}
-          private-key: ${{ secrets.APP_PRIVATE_KEY }}
-
-      - name: "Store GitHub token in environment"
-        run: echo "GH_TOKEN=${{ steps.app-token.outputs.token }}" >> "$GITHUB_ENV"
-        shell: bash
-
-      - uses: actions/checkout@v4
```

Comment:
Noted that this lines up with [secure-checkout/action.yml](https://github.com/mongodb-labs/drivers-github-tools/blob/7edc545fcf00ff1da4910079b7488c81d7c583b0/secure-checkout/action.yml).